contracts/contracts/coordination/Coordinator.sol

```diff
@@ -401,17 +407,26 @@ contract Coordinator is AccessControlDefaultAdminRules, FlatRateFeeModel {
         return getParticipantFromProvider(rituals[ritualId], provider);
     }
 
+    function isEncryptionAuthorized(
```

Comment:
Should the client call this method before attempting to encrypt? Or is it only meant to be called by Ursulas?

Reply:
It is meant to be called by Ursula.  This is convenient so that Ursula does not need to handle call forwarding and interface ABIs.

Reply:
Just for Ursulas, particularly since one of the parameters is the ciphertext digest, so the encryptor could only call it after the fact. But you're comment is a good one; the encryptor can call this method in the GlobalAllowList contract:
https://github.com/nucypher/nucypher-contracts/blob/74ac5eaf14a40e57652072531eae7c7f90bcfa1a/contracts/contracts/coordination/GlobalAllowList.sol#L38-L40

Note that this method is specific to `GlobalAllowList`, and not to the `IEncryptionAuthorizer` interface. I think we probably need something there as well.

Reply:
Opened #118